tsconfig.json

```diff
@@ -1,26 +1,22 @@
 {
   "compilerOptions": {
-    "target": "ESNext",           
-    "module": "commonjs",                 
-    "lib": ["es6", "es2015", "dom"], 
-    "declaration": false,     
+    "target": "ESNext",
+    "module": "commonjs",
+    "lib": ["es6", "es2015", "dom"],
+    "declaration": true,
```

Comment:
This can be changed to false because only npm installers could "benefit", but it makes cgx project a lot larger.